scripts/_proto/Dockerfile

```diff
@@ -12,4 +12,4 @@ RUN uv add "protobuf<4"
 RUN uv sync --all-extras
 RUN poe gen-protos
 
-CMD cp -r ./temporalio/api/* /api_new && cp -r ./temporalio/bridge/proto/* /bridge_new
+CMD ["sh", "-c", "cp -r ./temporalio/api/* /api_new && cp -r ./temporalio/bridge/proto/* /bridge_new"]
```

Comment:
Avoids warning:

```
     - JSONArgsRecommended: JSON arguments recommended for CMD to prevent
     unintended behavior related to OS signals (line 15)
```